tincparser/conf.py

```diff
@@ -51,7 +52,7 @@ def __init__(self, io, filename):
         self._open()
 
     def _parse(self, f):
-        result = TincConfParser.conf_file.parseFile(f)
+        result = TincConfParser.conf_file.parseString(to_unicode(f))
```

Comment:
As ``_parse`` does not parse a file anymore but a string, I would somehow make this visable somehow.

Please either change the parameter ``f`` (which sounds like "file") to something that sounds like "string" or "contents" or (also) add a docstring.

Reply:
Fixed in 76d2c36